lib/miq_expression.rb

```diff
@@ -977,23 +983,24 @@ def self.get_column_details(column_names, class_path, assoc_path, opts)
       excludes += ["^.*derived_host_count_off$", "^.*derived_host_count_on$", "^.*derived_vm_count_off$", "^.*derived_vm_count_on$", "^.*derived_storage.*$"]
     end
 
-    column_names.collect do|c|
+    column_names.collect do |c|
       # check for direct match first
       next if excludes.include?(c) && !EXCLUDE_EXCEPTIONS.include?(c)
 
       # check for regexp match if no direct match
       col = c
-      excludes.each do|excl|
-        if c.match(excl)
-          col = nil
-          break
+      unless EXCLUDE_EXCEPTIONS.include?(c)
```

Comment:
the original code had it as well, so I don't think you screwed it up, I was just looking at it without those blinders on 🕶